src/SelfGravity/Poi_UserWorkBeforePoisson_GREP.cpp

```diff
@@ -102,6 +101,9 @@ void Poi_UserWorkBeforePoisson_GREP( const double Time, const int lv )
             Aux_Error( ERROR_INFO, "GREP_CENTER_COM has not been implemented yet!!\n" );
          break;
 
+         case GREP_CENTER_RESTART: // GREP center from RESTART data
```

Comment:
Update the comment to `restart file`.

Reply:
Changed this to `GREP_CENTER_NONE` and moved above.

Reply:
Update the comment of `GREP_CENTER_NONE`. In fact, the `GREP_Center` is unchanged in this case, instead of not assigned. Also, it is clear that switch block is for the GREP center, and thus I think it is unnecessary to mention it again in the comment.

Reply:
I have just removed the comment.

Reply:
So you think that the name `GREP_CENTER_NONE` is clear and does not require any comments? I meant removing the beginning term "GREP center" in the comment, rather than all the comments.

Reply:
What exact comment do you want?

Reply:
`restart file`

Reply:
I updated the comment.